hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.test.js

```diff
@@ -142,6 +150,21 @@ describe("Connected tables", () => {
         );
     });
 
+    it("Calls reportIncident when the 'Report broken/lost' button is clicked", () => {
+        const oneRow = [
```

Comment:
Can you take one of the items from the mocked data instead? That way if we change the structure of the API response, we just need to update the mock data to see what breaks (given we lack integration tests at the moment)